app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+def get_file_content():
+    message = 'Enter new line of content: '
+    content = []
+    while (line := input(message)) != 'stop':
+        content.append(line + '\n')
+    return content
+
+
+def create_file():
+    file_name = input('Enter name of the file:')
```

Comment:
```suggestion
    file_name = input('Enter name of the file: ')
```

Reply:
It's better to add space. Otherwise, the name of the file and this message aren't separated.